src/pkg/term/table.go

```diff
@@ -30,8 +30,12 @@ func (t *Term) Table(slice interface{}, attributes ...string) error {
 	}
 
 	// Print headers
-	for _, attr := range attributes {
-		_, err = fmt.Fprintf(w, "%s\t", attr)
+	for i, attr := range attributes {
+		var prefix string
+		if i > 0 {
+			prefix = "\t"
+		}
+		_, err = fmt.Fprint(w, prefix, attr)
 		if err != nil {
 			return err
 		}
```

Comment:
```suggestion
	header := strings.Join(attributes, "\t")
	_, err = fmt.Fprint(w, header)
	if err != nil {
		return err
	}
```

There's a "}" after this which I couldn't select but should be deleted if you go with this code suggestion

Reply:
Thanks, good one